### ISSUE TITLE
Original youtube videoids

### ISSUE THREAD

**Issue by m-bain** [ROC-HCI/UR-FUNNY#4]
Hi,

Would you be able to share the original youtube videoids? We are pre-training on extra data and want to ensure there is no overlap with your test set.

Max

**Comment by matalvepu** [comment]
You can find the unique video number in the language_sdk.pkl. There is one field named "root" which denotes the unique video number from where the instance was extracted. You can use it to verify if there is any overlap. However, I made sure that there is no overlap among train, dev, and test set.
